content/bio.md

```diff
@@ -0,0 +1,14 @@
+# Kris Nóva Biography
+
+_July 2021_
+_Updated for Twilio, Personal, and The Privilege Escalation Foundation_
+
+Kris Nóva is a computer scientist, alpinist, author, public speaker and transgender advocate best known for her work on Linux and Kubernetes. 
+She specalizes in Linux kernel security, distributed systems and infrastructure management, and open source software engineering. 
+In 2017 she co-authored Cloud Native Infrastructure.
+
+She is well known for her open source contributions including projects like Linux, Kubernetes, and The Go Programming Language. 
+She is a popular public speaker, best known for her Kubernetes clusterfuck talks. She co-founded the cluster API project, created Kubicorn, and created naml for managing Kubernetes resources with pure Go.
+
+Today she works as a Senior Principal engineer building enterprise bare metal Kubernetes systems. 
+She is the founder of The Prvilege Escalation Foundation which is a non-profit focusing on offering resources for transgender minds in STEM.
```

Comment:
s/Prvilege/Privilege

Reply:
she knew that what she made would be the finest in the nation

Reply:
👜  🏷️  💸 🔪 🏬